src/wallet/rpcwallet.cpp

```diff
@@ -2553,6 +2561,19 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
             feeRate = CFeeRate(AmountFromValue(options["feeRate"]));
             overrideEstimatedFeerate = true;
         }
+
+        if (options.exists("subtractFeeFromAmount")) {
+            subtractFeeFromAmount = options["subtractFeeFromAmount"].get_array();
+            for (unsigned int idx = 0; idx < subtractFeeFromAmount.size(); idx++) {
+                string strAddress(subtractFeeFromAmount[idx].get_str());
+                CBitcoinAddress address(strAddress);
+                if (!address.IsValid())
+                    throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, string("Invalid Bitcoin address: ")+strAddress);
+                if (setSubtractFeeFromAmount.count(strAddress))
+                    throw JSONRPCError(RPC_INVALID_PARAMETER, string("Invalid parameter, duplicated address: ")+strAddress);
+                setSubtractFeeFromAmount.insert(strAddress);
```

Comment:
@ryanofsky I like the idea but am a bit worried about the interaction of `subtractFeeFromPositions` and `changePosition`. It might not be clear to the user if the position marking is done before or after change output is added, or discount the wrong output by adding a `changePosition` argument.